content/zh-cn/releases/patch-releases.md

```diff
@@ -127,6 +129,8 @@ dates for simplicity (every month has it).
 在两个月的维护模式期间，发布管理员可能会删减额外的维护版本以解决：
 
 - CVE（在安全响应委员会的建议下）
+- [漏洞](/zh-cn/docs/reference/issues-security/official-cve-feed/)
+  已分配 CVE ID 的漏洞（在安全响应委员会的建议下）。
```

Comment:
From the [preview](https://deploy-preview-48583--kubernetes-io-main-staging.netlify.app/zh-cn/releases/patch-releases/), it is suggested to change to:
```suggestion
- 已分配 CVE ID 的[漏洞](/zh-cn/docs/reference/issues-security/official-cve-feed/)（在安全响应委员会的建议下）。
```